components/bl-social-login-button-component/component.json

```diff
@@ -0,0 +1,150 @@
+{
+  "id": "c_77c399a94abeab643336805b91a3aefb",
+  "name": "Social Login Button",
+  "description": "The Social Login Button component is designed for quick user login.",
+  "showInToolbox": true,
+  "faIcon": "toggle-on",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "type": "checkbox",
+      "name": "disabled",
+      "label": "Disabled",
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "handlerId": "disabledLogic",
+      "settingTooltip": "This is the handler to disable the component.",
+      "dataBinding": true,
+      "handlerDescription": "This is the handler to disable the component.",
+      "handlerLabel": "Disabled Logic"
+    },
+    {
+      "type": "text",
+      "name": "redirectToPage",
+      "label": "Redirect To Page",
+      "settingTooltip": "This handler allows you to specify which page to go to after login.",
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "handlerId": "redirectToPageLogic",
+      "handlerLabel": "Redirect To Page Logic",
+      "handlerDescription": "This handler allows you to specify which page to go to after login.",
+      "dataBinding": true
+    },
+    {
+      "type": "text",
+      "name": "extraQueryParams",
+      "label": "Extra Query Params",
+      "settingTooltip": "This handler allows you to add extra query parameters that will be added to the URL address.",
+      "showInSettings": true,
+      "defaultValue": "",
+      "hasLogicHandler": true,
+      "handlerId": "extraQueryParamsLogic",
+      "handlerLabel": "Extra Query Params Logic",
+      "handlerDescription": "This handler allows you to add extra query parameters that will be added to the URL address.",
+      "dataBinding": true
+    },
+    {
+      "type": "checkbox",
+      "name": "iconsVisibility",
+      "label": "Icons Visibility",
+      "showInSettings": true,
+      "defaultValue": true,
+      "hasLogicHandler": true,
+      "handlerId": "iconsVisibilityLogic",
+      "handlerLabel": "Icons Visibility Logic",
+      "dataBinding": true,
+      "settingTooltip": "This handler allows you to control the visibility of the button icon.",
+      "handlerDescription": "This handler allows you to control the visibility of the button icon."
+    },
+    {
+      "type": "checkbox",
+      "name": "googleplus",
```

Comment:
@v-excelsior @Saharij wouldn't it be better to have a Select to choose a login provider instead of these checkboxes?

the component name is `Social Login Button`, so if I were a customer I would expect the component add only one button

does it make sense?

Reply:
or, rename the component to Social Login Buttons

Reply:
I prefer the second variant.

Reply:
ok, @Saharij please rename the directory/styles/docs/etc as well

Reply:
still `bl-social-login-button-component`